argo-cd-apps/overlays/production-downstream/delete-applications.yaml

```
@@ -29,3 +29,15 @@ kind: ApplicationSet
 metadata:
   name: quality-dashboard
 $patch: delete
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: workspaces
+$patch: delete
```

@manish-jangra this looks like a bad rebase, we should not be deleting workspaces in this PR.
```suggestion
---
```

---

@hugares It seems that main branch doesn't have delete entry for workspaces component. [Main Branch](https://github.com/redhat-appstudio/infra-deployments/blob/main/argo-cd-apps/overlays/konflux-public-production/delete-applications.yaml)